### PR TITLE
New_3_init-postgresdb_Smetanin

### PR DESCRIPTION
**Pull request by dmytrosmetanin**
new
initialized PostgresDB

### CODE REVIEW

File: codeArena/codeArena/settings.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+# Database
+# https://docs.djangoproject.com/en/3.2/ref/settings/#databases
+
+DATABASES = {
+
+    'default': {
+
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+
+        'NAME': config('DB_NAME'),
```
Comment (vvelizariy) [inline codeArena/codeArena/settings.py:87]: Please also use default values with type casting if needed, like
PORT = config('DB_PORT', default=5050, cast=int)
PG_HOST= config('PG_HOST', default='127.0.0.1')

Comment (vvelizariy) [reply codeArena/codeArena/settings.py:87]: Like in examples here
https://pypi.org/project/python-decouple/

File: codeArena/codeArena/settings.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+# https://docs.djangoproject.com/en/3.2/howto/static-files/
+
+STATIC_URL = '/static/'
+
+# Default primary key field type
+# https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
+
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+ALLOWED_HOSTS = ['127.0.0.1']
```
Comment (vvelizariy) [inline codeArena/codeArena/settings.py]: Dont forget empty newlines in the end of file

File: codeArena/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+asgiref==3.4.1
```
Comment (vvelizariy) [inline codeArena/requirements.txt]: Duplicate file, should be only one in root repository folder

**Review by vvelizariy — COMMENTED** [review]
Please add to .gitignore
*.pyc
to avoid commiting .pyc files

File: codeArena/codeArena/settings.py [hunk 4]
```diff
@@ -80,14 +81,33 @@
...
+
+
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+
+        'NAME': config('DB_NAME'),
+
+        'USER': config('DB_USER'),
+
+        'PASSWORD': config('DB_PASSWORD'),
+
```
Comment (vvelizariy) [inline codeArena/codeArena/settings.py]: Please remove blank lines, to keep same style as mongodb is declared

File: codeArena/codeArena/settings.py [hunk 5]
```diff
@@ -80,14 +81,33 @@
 MONGODB_HOST = config("MONGODB_HOST")
```
Comment (vvelizariy) [inline codeArena/codeArena/settings.py:81]: Please use also default value for parameter, as 127.0.0.1 as you would anyway use Docker

File: codeArena/codeArena/settings.py [hunk 6]
```diff
@@ -80,14 +81,33 @@
...
 
 DATABASES = {
+
     'default': {
+
+
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+
+        'NAME': config('DB_NAME'),
```
Comment (vvelizariy) [inline codeArena/codeArena/settings.py:87]: suggestt also use default value as `codearena` if not any was exported.

**Review by MrSancho42 — APPROVED** [review]

**Review by MetalMari — APPROVED** [review]
Approved

**Review by Viktorika-p — APPROVED** [review]

**Pull request merged**